server/src/main/java/org/opensearch/index/get/GetStats.java

```diff
@@ -137,6 +138,7 @@ public long current() {
     public XContentBuilder toXContent(XContentBuilder builder, Params params) throws IOException {
         builder.startObject(Fields.GET);
         builder.field(Fields.TOTAL, getCount());
+        builder.field(Fields.GET_TIME, Objects.toString(getTime()));
```

Comment:
@hye-on I am very sorry, missed that, I think we should use `humanReadableField` here, the same way the `Fields.TIME` was serialized before (no need to use `Objects.toString`):

```suggestion
        builder.humanReadableField(Fields.TIME_IN_MILLIS, Fields.GET_TIME, getTime());
```

Reply:
@reta 
Initially, I tried using `humanReadableField`, but this method added both the first and second arguments, causing a duplicate error with `TIME_IN_MILLIS`.
To include only `getTime`, I checked the implementation of humanReadableField and found that it uses `Objects.toString(value)` for serialization.
For reference, calling `toString `on a `TimeValue` includes units like `s`or `ms`. 

#### Error details:
java.lang.RuntimeException: Failure at [indices.stats/60_time_fields.yml:11]: Duplicate field 'time_in_millis' at [Source: REDACTED (StreamReadFeature.INCLUDE_SOURCE_IN_LOCATION disabled); line: 1, column: 150].

#### humanReadableField
```java
public XContentBuilder humanReadableField(String rawFieldName, String readableFieldName, Object value) throws IOException {
    assert rawFieldName.equals(readableFieldName) == false : "expected raw and readable field names to differ, but they were both: "
        + rawFieldName;
    if (humanReadable) {
        field(readableFieldName, Objects.toString(value)); // Adds the human-readable field using Objects.toString(value)
    }
    HumanReadableTransformer transformer = HUMAN_READABLE_TRANSFORMERS.get(value.getClass());
    if (transformer != null) {
        Object rawValue = transformer.rawValue(value);
        field(rawFieldName, rawValue); // Adds the raw field transformed 
    } else {
        throw new IllegalArgumentException("no raw transformer found for class " + value.getClass());
    }
    return this;
}

```

Reply:
Ah, you are totally right, thank you